src/atwho.js

```diff
@@ -70,11 +70,11 @@ define([
             insertTpl: '${name}',
             property: '',
             outputValue: false,
-            useRichText: false,
+            noRichText: true,
             functionOverrides: {},
         });
 
-        if (options.useRichText) {
+        if (!options.noRichText) {
```

Comment:
not really a fan of the double-negative, but I suppose it was necessary because of default value, right?